app/services/clients/zendesk.client.js

```diff
@@ -1,33 +1,42 @@
 'use strict'
 
-const zendesk = require('node-zendesk')
+const axios = require('axios')
 const zendeskConfig = require('../../../config/zendesk')
 
-const zendeskClient = zendesk.createClient({
-  username: process.env.ZENDESK_USER,
-  token: process.env.ZENDESK_API_KEY,
-  remoteUri: process.env.ZENDESK_URL,
-  proxy: process.env.http_proxy
-})
+async function createTicket (opts) {
```

Comment:
As this method isn't currently waiting for anything or handling any errors it could be simplified a bit
```js
function createTicket(opts) {
  const zendeskTicketData = {
    // ...
  }
  return axios({
    method: 'post',
    data: zendeskTicketData,
    // ...
  })
}
```